### PR TITLE
E UI 4777 additional facilities display and function story

### PR DESCRIPTION
**Pull request by waynepHMCTS**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-4777

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.html [hunk 1]
```diff
@@ -1,14 +1,69 @@
...
+        Do you require any additional facilities?
+      </h1> 
+      <exui-case-flags
+        [info]="'Any active flags on this case may require additional facilities at the hearing.'"
+        [hearingValueModel]="serviceHearingValuesModel"
+        [caseFlagsRefData]="caseFlagsRefData"
+        [caseFlagType]="caseFlagType">
+      </exui-case-flags>
+    </div>
+    <div class="govuk-grid-column-two-thirds">
```
Comment (rayliangatos) [inline src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.html:16]: no need to define different grid width here, can reuse line 4
<div class="govuk-grid-column-three-quarters">

File: src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.html [hunk 2]
```diff
@@ -1,14 +1,69 @@
...
+      <div class="govuk-form-group govuk-!-margin-top-4" [ngClass]="{'govuk-form-group--error': !additionSecurityRequiredValid}">
+          <fieldset class="govuk-fieldset"  id="addition-security-confirmation">
+              <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+                  <h3 class="govuk-fieldset__heading">
+                      Will additional security be required?      &nbsp;  &nbsp; &nbsp; &nbsp;
```
Comment (rayliangatos) [inline src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.html:21]: are these random &nbsp; needed?

File: src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.spec.ts [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormArray, FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { provideMockStore } from '@ngrx/store/testing';
+import { of } from 'rxjs';
+import { ACTION } from '../../../models/hearings.enum';
+import { HearingsService } from '../../../services/hearings.service';
+import { initialState, serviceHearingValuesModel } from '../hearing.store.state.test';
+import { HearingFacilitiesComponent } from './hearing-facilities.component';
+import { RefDataModel } from 'src/hearings/models/refData.model';
```
Comment (rayliangatos) [inline src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.spec.ts:11]: src

File: src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.ts [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
-import {ActivatedRoute} from '@angular/router';
-import {Store} from '@ngrx/store';
-import {CaseFlagReferenceModel} from '../../../models/caseFlagReference.model';
-import {ACTION, CaseFlagType} from '../../../models/hearings.enum';
-import {HearingsService} from '../../../services/hearings.service';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormArray, FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { RefDataModel } from 'src/hearings/models/refData.model';
```
Comment (rayliangatos) [inline src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.ts:5]: src

File: src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.html (hunk 2 above)
Comment (waynepHMCTS) [reply src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.html:21]: Could please just say you what you want me to do? How can they be random. They were taken from design. @rayliangatos  what is your stance on using @nbspc I have seen them in @Kartheek-S work and nothing was said. We have closing the sprint. We do not have time for this @rayliangatos.

Comment (rayliangatos) [reply src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.html]: nbsp is only needed when it's needed, but this case it doesn't, though it's in the prototype we should remove it.

File: src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.html (hunk 1 above)
Comment (waynepHMCTS) [reply src/hearings/containers/request-hearing/hearing-facilities/hearing-facilities.component.html:16]: This again is the prototype design. Will reuse as stated.

**Review by rayliangatos — APPROVED** [review]

**Pull request merged**